src/nomad_json_parser/__init__.py

```diff
@@ -0,0 +1,97 @@
+from nomad.config.models.plugins import (
+    ParserEntryPoint,
+    SchemaPackageEntryPoint,
+)
+from pydantic import Field
+
+
+class JsonMapperEntryPoint(SchemaPackageEntryPoint):
+    parameter: int = Field(0, description='Custom configuration parameter')
+
+    def load(self):
+        from nomad_json_parser.schema_packages.jsonimport import m_package
+
+        return m_package
+
+
+jsonmapper_entry = JsonMapperEntryPoint(
```

Comment:
I would change this name for clarification, currently it's quite hard to figure which entry point is what in the `pyproject.toml`. Maybe:
```suggestion
json_mapper_schema_package = JsonMapperEntryPoint(
```